Valeriia/Movies_database_project/src/App.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { Component } from "react";
+import Header from "./components/Header";
+import Footer from "./components/Footer";
+import Movies from "./components/Movies";
+import SortPannel from "./components/SortPannel";
+import Loader from "./components/Loader";
+import movies from "./movies.json";
+import NotFound from "./components/NotFound";
+import Movie from "./components/Movie";
+import { IMovie } from "./types";
+import "./App.scss";
+
+interface IAppState {
+  movies: IMovie[] | [];
+  isLoading: boolean;
+  tempListOfMovies: IMovie[] | [];
+  filterBy: string;
+  sortBy: string;
+  searchTerm: string;
+  movie?: IMovie;
+}
+
+class App extends Component<{}, IAppState> {
```

Comment:
You should use App just for wrapping your Pages not for being a Page.
I wanna see just two Components(pages) inside of the App.